llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -670,6 +670,27 @@ static Error handleArgs(const CommonConfig &Config, const ELFConfig &ELFConfig,
     }
   }
 
+  if (Config.ChangeSectionLMAValAll != 0) {
+    for (Segment &Seg : Obj.segments()) {
+      if (Seg.FileSize > 0) {
+        if (Config.ChangeSectionLMAValAll > 0 &&
+            Seg.PAddr > std::numeric_limits<uint64_t>::max() -
+                            Config.ChangeSectionLMAValAll) {
+          return createStringError(errc::invalid_argument,
+                                   "address 0x" + Twine::utohexstr(Seg.PAddr) +
```

Comment:
For this and the similar error below, I think it would be useful to give a little more context to the error, something like "would overflow when adjusting by <value>".

Reply:
Improved this as well.